native/src/components/MapView.tsx

```diff
@@ -69,7 +84,6 @@ const MapView = ({ boundingBox, featureCollection, currentFeature }: MapViewProp
 
   const onLocationPermissionRequest = useCallback((locationPermission: PermissionStatus | undefined) => {
     const permissionGranted = locationPermission === RESULTS.GRANTED
-    setFollowUserLocation(permissionGranted)
```

Comment:
Why did you remove this?

Reply:
because it overrides the selectedFeature coordinates if you redirect from pageDetail. So everytime you enter mapView you follow automatically userLocation. This shouldn't be the case